projects/helper/chains.json

```diff
@@ -169,6 +169,7 @@
   "liquidchain",
   "litecoin",
   "loop",
+  "lorenzo",
```

Comment:
lorenzo is a chain?

Reply:
Yeah. Lorenzo is an evm compatible chain with cosmos architecture.

Reply:
is the chain live? can you share rpc and explorer info?

Reply:
RPC:  https://rpc.lorenzo-protocol.xyz
ChainId: 8329

Reply:
also, any protocol already live? we usually list a chain only where there is a live defi project in it